utils/locales.js

```diff
@@ -1,8 +1,346 @@
+const { SUPPORTED_GAMES } = require('./constants');
+const Config = require('../config.json');
+
 const LOCALES = {
 	en: {
```

Comment:
For a further release, I will separate each language in its own file.

Reply:
OK, let me know how you want to do that and if you need any help.

Reply:
I'll do it after this pull request is merged.
I will also propose you to join the project as a collaborator.

Reply:
Thank you, i'm happy to join the team :)